.stylelintrc

```diff
@@ -5,15 +5,24 @@
     "no-duplicate-selectors": null,
     "declaration-colon-newline-after": null,
     "value-list-comma-newline-after": null,
-    "property-no-unknown": [true, {
```

Comment:
I'm not sure why but I was forced to run prettier on several files when I merged to master 🤔

Reply:
The Gatsby 4 update also updated linters alongside it, and apparently those linters either weren't as stringent or weren't running at all before the update. That said, I don't know why changes that weren't originally in this PR are getting added by Prettier, they should have been taken care of by the retroactive fixes I did in that PR.